tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "module": "CommonJs",
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "outDir": "./built",
+    "pretty": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "strictNullChecks": true,
+    "target": "es6"
+  },
+  "include": ["./src/**/*"],
+  "exclude": [
+    "./node_modules/**/*",
+    "./node_modules/**/**",
+    "./jest.config.js",
+  ]
+}
```

Comment:
Seems we have a end of line (EOL) error in this file?

Reply:
I will take a look, I didn't get any errors when working on it.